JS/Lesson19/tugba/exercises.js

```diff
@@ -1,8 +1,162 @@
 // 1. Create a Person class with name, age, and gender properties, and a greet() method that logs a greeting message to the console. Then create a Student class that extends the Person class and adds a major property. Override the greet() method in the Student class to log a different greeting message to the console. Create 2 variables, one of type Person and one of type Student and log them to console.
+// I HAVE TO BE HONEST. CHAT GPT MY BEST FRIEND. I WOULDN'T DO ANYTHING WITHOUT IT.
```

Comment:
😂 Honesty is always good :) I'm fine with you using Chat GPT as long as you understand every line of code it gives you :)